File: 0771-jewels-and-stones/0771-jewels-and-stones.java
class Solution {
     public int numJewelsInStones(String jewels, String stones) {
        List<Character> jewelsList = new ArrayList<>();
        char[] arr = jewels.toCharArray();
        for (char c : arr) {
            jewelsList.add(c);
        }
        char[] arr2 = stones.toCharArray();
        int stoneJewelCount = 0;
        for (char d : arr2) {
            if (jewelsList.contains(d)) {
                stoneJewelCount++;
            }
        }
        return stoneJewelCount;
    }
}

File: 0771-jewels-and-stones/0771-jewels-and-stones.cs
public class Solution {
    public int NumJewelsInStones(string jewels, string stones)
        {
            List<char> list = new List<char>();
            list.AddRange(jewels);
            int count = 0;

            char[] arr = stones.ToCharArray();
            foreach (char c in arr)
            {
                if (list.Contains(c)) {
                    count++;
                }
            }

            return count;
        }
}